pkg/standalone/standalone.go

```diff
@@ -191,7 +194,7 @@ func isContainerRunError(err error) bool {
 	return false
 }
 
-func runPlacementService(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string) {
+func runPlacementService(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string, installLocation string) {
```

Comment:
This function doesnt amek use of installLocation, so param can be removed.

Reply:
Unfortunately this is needed to fit the signature of the `initSteps` func.